### ISSUE TITLE
Version selector is not displayed correctly after enabling showing version alias

### ISSUE THREAD

**Issue by raymond-u** [squidfunk/mkdocs-material#7187]
### Bug description

After upgrading to the latest version and enabling showing version alias, the version selector starts to displays some extra character.

Before:

<img width="91" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/36328498/c11c84d1-45d4-4b44-9398-9957c631c974">

Now:

<img width="114" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/36328498/a072c70d-60d7-45bd-a320-91bd5633726b">

Related part of `mkdocs.yaml`:

```
extra:
  version:
    provider: mike
    alias: true
    default:
      - nightly
      - stable
```

The command I use in GitHub Actions to deploy docs:

```
# Docs for nightly version
poetry run mike deploy --push nightly

# Docs for stable version
poetry run mike deploy --push --update-aliases ${{ steps.docs_version.outputs.docs_version }} stable
poetry run mike set-default --push stable
```

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- https://github.com/squidfunk/mkdocs-material/issues/6436

### Reproduction

N/A

### Steps to reproduce

N/A

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [ ] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. Unfortunately, without a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), it's unclear what is causing the problem. If you could at least share your `versions.json` that is built by mike, it would be a start.

**Comment by raymond-u** [comment]
Apologies. Here is my `version.json`:

```
[
  {
    "version": "nightly",
    "title": "nightly",
    "aliases": []
  },
  {
    "version": "0.4",
    "title": "0.4",
    "aliases": [
      "stable"
    ]
  },
  {
    "version": "0.3",
    "title": "0.3",
    "aliases": []
  },
  {
    "version": "0.2",
    "title": "0.2",
    "aliases": []
  },
  {
    "version": "0.1",
    "title": "0.1",
    "aliases": []
  }
]
```

**Comment by squidfunk** [comment]
Thanks! Fixed in 407335d20.

**Comment by squidfunk** [comment]
Release as part of [9.5.24](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.24).
